quest_maker_app/models/user_quest.py

```diff
@@ -71,16 +73,20 @@ def get_info(self):
 
     def get_daily_info(self):
         """Get daily info for each day in the quest, reverse sorted by day"""
-        latest_day = self.latest_day
         info = []
-        for datum in self.user.dailydistance_set.filter(
-            day__range=[self.quest.start_date, latest_day]):
+        for day in util.daterange(self.quest.start_date, self.latest_day):
```

Comment:
This isn't great bc it hits the db so many times.  Any thoughts on how to do it better?  Basically, I want to get back info for all days in the quest, even if there isn't data in the db.  The most straight forward way to do that is to look for the data in the db for each day, then default to distance=0 if there is no data in the db.  As I said, not great.